docs/installation.md

```diff
@@ -35,49 +41,63 @@ This installation guide assumes that the following softwares are already install
 

```

Comment:
@sandoche2k When replying to a sub-thread like this one, do you think you could add your comment to the actual sub-thread rather than the main thread of the PR? That way we can keep our parallel discussions separated.